### ISSUE TITLE
Misflagged search application as game

### ISSUE THREAD

**Issue by ebergstein** [UncleGoogle/galaxy-integration-humblebundle#104]
**Describe the bug**
I installed the plugin from Galaxy, and it flagged Everything(the search application found at https://www.voidtools.com/) as Everything(the video game) and thinks that the game is installed from Humble Bundle.  Launching it will launch the search application.

**Expected behavior**
I was expecting it to show that the Humble Bundle version of Everything is not installed, since I do not have it installed.

**Plugin installed**
directly from Galaxy

**Provide logs**
- Install the search application, do not install the game, and install the plugin.  It should show that the Humble Bundle version of Everything is installed if you have the humble version in your library.
The one file is all I have for the plugin.
[plugin-humble-f0ca3d80-a432-4d35-a9e3-60f27161ac3a.zip](https://github.com/UncleGoogle/galaxy-integration-humblebundle/files/4562106/plugin-humble-f0ca3d80-a432-4d35-a9e3-60f27161ac3a.zip)

**Comment by UncleGoogle** [comment]
Hi, Thanks for the report.

Well, that's the result of heuristics used to find installed games based on owned game names.

The only way I see is to provide more control on what is installed via GUI. Definitely to be done in v0.9 or 0.10 ...

**Comment by nodeunit** [comment]
Funnily enough, I have this issue as well. 

Does Galaxy provide any options for hiding a particular integration's installation (like hiding a game, but only for, say, the Humble Bundle version)?
Alternatively, perhaps as a feature, managing installations/installation paths of games through this particular integration? Or just adding an 'ignore' list for valid paths?

**Comment by UncleGoogle** [comment]
No such option in Galaxy.

Yes, I've planned to add some kind of config. 
Maybe an option to mark particular exe path as invalid is good enough. Then in GUI you can reject some auto-matching proposals. I think that would give enough flexibility without breaking change for autodetection. What do you think?

**Comment by nodeunit** [comment]
> Yes, I've planned to add some kind of config.
> Maybe an option to mark particular exe path as invalid is good enough. Then in GUI you can reject some auto-matching proposals. I think that would give enough flexibility without breaking change for autodetection. What do you think?

Yes, the latter option would be a good fit for the time being, until sometime in the future a more full installation flow is possible.

**Comment by UncleGoogle** [comment]
> until sometime in the future a more full installation flow is possible.

@nodeunit do you have any idea about full installation flow, how it should look like etc? How it can influence installed games detection?
